deploy.ts

```diff
@@ -119,3 +130,21 @@ void (async () => {
     process.exit(1);
   }
 })();
+
+function prepareUploadOptions() {
+  const options: CustomUploadOptions = {};
```

Comment:
Not sure why this needs to be a type aside from a generic object? CustomUploadOptions isn't even from the same library as the nodejs client it gets used on.